docs/guides/security-policy-for-authors.md

```diff
@@ -327,6 +330,9 @@ For example, see [\[2\]](#references-and-notes) (Perl-Sec-Policy).
 
 ## Examples
 
+These examples below are in the public domain.
```

Comment:
Mention CC0? https://creativecommons.org/public-domain/cc0/

Reply:
No need, mentioning "public domain" directly is even better

Reply:
I guess that I could reference CC0 in parens.

Reply:
OK

Reply:
I've added a link around "public domain" to CC0.